.github/workflows/rebase-upstream.yaml

```
@@ -18,6 +18,7 @@ permissions:
   contents: write
   pull-requests: write
   issues: write
+  workflows: write
```

actionlint (1.7.8)

unknown permission scope "workflows". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)